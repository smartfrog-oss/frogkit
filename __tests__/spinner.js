import { mount } from 'vue-test-utils'
import Spinner from '@/components/spinner'


describe('Spinner component', () => {
  it('should render component and match snapshot', () => {
    const wrapper = mount(Spinner)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
