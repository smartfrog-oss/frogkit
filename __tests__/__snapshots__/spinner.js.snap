// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner component should render component and match snapshot 1`] = `undefined`;
